3rdParty/iresearch/external/simdcomp/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ else()
 endif()
 
 # test for arm
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64.*|AARCH64.*)")
```

Comment:
Note that this is probably woefully inadequate to check whether the CPU at hand even supports NEON (or some other SIMD extension).

Reply:
@markuspf in order to persist any change in `3rdparty/iresearch/` there is a need to make a PR to https://github.com/iresearch-toolkit/iresearch/. Otherwise it could be erased during the next update.

Reply:
How long will it take for such fixes to come through to arangodb?

Reply:
https://github.com/iresearch-toolkit/iresearch/pull/314